src/bindings/glib/gelektra-key.c

```diff
@@ -169,6 +169,39 @@ GElektraKey *gelektra_key_gi_make(GElektraKey *key)
 	return (key != NULL) ? gelektra_key_make(key->key) : NULL;
 }
 
+/* initialization */
+static void gelektra_key_gi_init_va(GElektraKey *key, const gchar *name, ...)
+{
+	if (!name)
+		return;
+	va_list va;
+	va_start(va, name);
+	keyVInit(key->key, name, va);
+	keyIncRef(key->key); // keyVInit will clear the refcount
```

Comment:
glib/GI bindings call gelektra_key_init upon gobject creation. gelektra_key_init calls keyNew which will set the refcount to 1. Afterwards keyVInit get's call which will clear our refcount. So we have to increase it again.
So the comment make sense.